.github/workflows/ci.yml

```diff
@@ -13,10 +13,12 @@ jobs:
     strategy:
       matrix:
         python-version: [3.7, 3.8, 3.9]
-        django-version: [22, 32, 40]
+        django-version: [22, 32, 40, 41]
         exclude:
           - python-version: 3.7
             django-version: 40
+          - python-version: 3.7
```

Comment:
Maybe we should move to python 3.10 as the default, and add it to the matrix too. I'm not sure if 3.7 support is still warranted, but I'm fine keeping it for now.

Reply:
I'm not sure how django-ses selects the default Python, but I'm happy to change it. I've added it to the gitlab test suite.

> I'm not sure if 3.7 support is still warranted, but I'm fine keeping it for now.

Yeah I wanted to be conservative and I didn't need any changes that would require bumping the version. I'm happy to remove 3.7 from tests and pypi classifiers if you'd like.